src/js/user-profile/components/AuthApplicationSection.jsx

```diff
@@ -1,18 +1,26 @@
+import PropTypes from 'prop-types';
 import React from 'react';
-import { connect } from 'react-redux';
-
-import { handleVerify } from '../../common/helpers/login-helpers.js';
 
 class AuthApplicationSection extends React.Component {
   constructor(props) {
     super(props);
-    this.handleVerify = handleVerify;
+    this.handleVerify = this.handleVerify.bind(this);
+  }
+
+  handleVerify() {
```

Comment:
We chatted about this by phone, but replying here for the logs. I believe the old `handleVerify` was in the shared `login-helpers.js` code precisely for that reason (same code needed 2 places), but that handler was doing 2 things: fetching the url and opening the popup window.

I guess we could keep it here but we would still need to update the component prop with the value returned by the fetch. I guess the shared code could fire an action and any component that needed the url could update itself in its reducer. OK, will try this.